mini-lsm-book/src/02-sst.md

```diff
@@ -39,6 +44,17 @@ more data than meta block, we can simply return the size of data blocks for `est
 You can also align blocks to 4KB boundary so as to make it possible to do direct I/O in the future. This is an optional
 optimization.
 
+The recommend sequence to finish **Task 1** is as below:
+
+- Implement `SsTableBuilder` in `src/table/builder.rs`
+  - Before implementing `SsTableBuilder`, you may want to take a look in `src/table.rs`, for `FileObject` & `BlockMeta`.
+  - For `FileObject`, you should at least implement `read`, `size` and `create`(No need for Disk I/O) before day 4.
+  - For `BlockMeta`, you may want to add some extra fields when encoding / decoding the `BlockMeta` to / from a buffer.
+- Implement `SsTable` in `src/table.rs`
+  - Same as above, you don't need to worry about `BlockCache` until day 4.
```

Comment:
```suggestion
  - Same as above, you do not need to worry about `BlockCache` until day 4.
```

don't -> do not looks more formal? I forgot if I was sticking to this writing standard when I was writing this, but it is good to enforce that afterwards 🤪